docs/safemode.md

```diff
@@ -0,0 +1,55 @@
+# Safemode
+
+Safemode represents a number of safety nets we have implemented into the chaos controller to help new and experienced users feel more confident deploying new disruptions to their environments.
+
+The Chaos Controller can be scary to use in production environments, but we learn a lot more running chaos experiments in our production environments. Considering this, having safety nets in place makes the entire process of running chaos experiments in high value environments a little safer.
+
+Safemoe will be added true by default and will require manual disabling of safety nets in order to bypass. In the disruption schema we have `unsafeMode` which represents ways to remove the safety nets.
+`disableAll` is self-explainatory while the other disables in the unsafe schema represent each safety net which you may disable one by one.
+Please take a look at the example below to see how to use the `unsafeMode` parameter.
+
+## Ignoring Safety Nets
+
+Because the list of safety nets to be implemented will grow in the future, there will surely be overlap with safety nets which will make it difficult for a user who is confident a specific safety net is not necessary but unsure if others will be.
+Therefore the controller allows for you to disable specific safety nets in the Safemode Spec. Checkout out example below to see how to remove certain safety nets.
+Keep in mind that all safety nets are turned on by default.
+
+### Safety Nets
+
+| Safety Net                                                | Type | Description                                                                                                                                               | IgnoreName                 |
+|-----------------------------------------------------------| ----------- |-----------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------|
+| Namespace-wide Targeting                                  | Generic | Using generical label selectors (e.x. X) that selects a majority of pods/nodes in a namespace as a target to inject a disruption into                     | DisableCountTooLarge       |
```

Comment:
```suggestion
| Namespace-wide Targeting                                  | Generic | Using generic label selectors that select a majority of pods/nodes in a namespace as a target to inject a disruption into                     | DisableCountTooLarge       |
```

Could we specify the exact percentage, rather than `majority`? (unless it is literally 50+%, then this is fine)

Reply:
Currently Ive added arbitrary values for the safety check:
80% at the Namespace level
66% at the Cluster level

Not sure what more appropriate values would be, have any ideas?

Reply:
The values are fine, I just feel we should tell the users the exact numbers we're checking for, here in the docs